lib/kafo/kafo_configure.rb

```diff
@@ -307,9 +311,9 @@ def set_app_options
                             :default => false
       self.class.app_option ['-s', '--skip-checks-i-know-better'], :flag, 'Skip all system checks', :default => false
       self.class.app_option ['--skip-puppet-version-check'], :flag, 'Skip check for compatible Puppet versions', :default => false
-      self.class.app_option ['-v', '--verbose'], :flag, 'Display log on STDOUT instead of progressbar'
-      self.class.app_option ['-l', '--verbose-log-level'], 'LEVEL', 'Log level for verbose mode output',
-                            :default => 'info'
+      self.class.app_option ['--progress-bar'], :flag, 'Display a progress bar for output', :default => false
+      self.class.app_option ['-l', '--verbose-log-level'], 'LEVEL', 'Log level for output',
```

Comment:
```suggestion
      self.class.app_option ['-l', '--output-log-level'], 'LEVEL', 'Log level for output',
```
Or perhaps `--terminal-log-level`. The old naming no longer makes sense here.